go/libraries/doltcore/sqle/database.go

```diff
@@ -314,6 +314,10 @@ func (db Database) GetTableInsensitiveAsOf(ctx *sql.Context, tableName string, a
 	}
 }
 
+func isDoltSystemTable(tableName string) bool {
+	return doltdb.HasDoltPrefix(tableName) || !resolve.UseSearchPath
```

Comment:
This seems off, won't these always both be true in the context?

Seems like maybe call to this method should just be replaced with `!resolve.UseSearchPath`, would be more explicit what was going on